pkg/apis/projectcalico/types.go

```diff
@@ -0,0 +1,272 @@
+// Copyright (c) 2019-2020 Tigera, Inc. All rights reserved.
+
+package projectcalico
+
+import (
+	"github.com/projectcalico/api/pkg/apis/projectcalico/v3/calico"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// NetworkPolicyList is a list of Policy objects.
+type NetworkPolicyList struct {
+	metav1.TypeMeta
+	metav1.ListMeta
+
+	Items []NetworkPolicy
+}
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+type NetworkPolicy struct {
+	metav1.TypeMeta
+	metav1.ObjectMeta
+
+	Spec calico.NetworkPolicySpec
+}
+
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// GlobalNetworkPolicyList is a list of Policy objects.
+type GlobalNetworkPolicyList struct {
+	metav1.TypeMeta
+	metav1.ListMeta
+
+	Items []GlobalNetworkPolicy
+}
+
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+type GlobalNetworkPolicy struct {
+	metav1.TypeMeta
+	metav1.ObjectMeta
+
+	Spec calico.GlobalNetworkPolicySpec
```

Comment:
Why are some of these importing the spec from `calico` and others (see GlobalNetworkSet) are not?